### ISSUE TITLE
Ensure public site pages have unique titles

### ISSUE THREAD

**Issue by emilyjablonski** [bloom-housing/bloom#5186]
Ensures each public page has a unique site title to fulfill [WCAG criterion 2.4.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-title.html)

QA Notes
If you navigate through [the public site](https://bloom-public-seeds.netlify.app/), all pages should have a custom title in the browser tab bar.
